packages/office-ui-fabric-react/src/utilities/color/hsv2hsl.ts

```diff
@@ -0,0 +1,16 @@
+import { IHSL } from './interfaces';
+import { MAX_COLOR_SATURATION, MAX_COLOR_VALUE } from './consts';
+
+/** Converts HSV components to an HSL color. */
+export function hsv2hsl(h: number, s: number, v: number): IHSL {
+  s /= MAX_COLOR_SATURATION;
+  v /= MAX_COLOR_VALUE;
+
+  let l = (2 - s) * v;
```

Comment:
For someone who doesn't know anything about color theory, what does all of this mean?

Reply:
I have no idea =)

All this code is copied and pasted, all I did is move it. Improvements can/should be made in a subsequent PR.

Reply:
As background, most of these functions were copied from the [color-functions package](https://www.npmjs.com/package/color-functions)...so none of us have any idea what this means either. :P I'm also not familiar with the background of why we copied the functions in rather than just referencing the package.